elastic/elastic-agent
# Add filestream ID on standalone kubernetes manifest

## What does this PR do?

This commit add unique IDs for the filestream inputs used by the
Kubernetes integration in the Elastic-Agent standalone
Kubernetes configuration/manifest file.

## Why is it important?

It fix a data duplication issue

## Checklist



~~## Author's Checklist~~
## How to test this PR locally
1. Use `deploy/kubernetes/elastic-agent-standalone-kubernetes.yaml` to deploy the Elastic-Agent on Kubernetes
2. Make sure there are multiple containers running
3. Restart the Elastic-Agent container a few times

There should be no data duplication, nor should be any logs from Filebeat like this:
```
filestream input with ID 'xxxxxx' already exists, this will lead to data duplication, please use a different ID
```

## Related issues

- Relates https://github.com/elastic/beats/issues/31512

~~## Use cases~~
~~## Screenshots~~
~~## Logs~~
